provider/googleProvider.js

```diff
@@ -89,6 +89,13 @@ class GoogleProvider {
         cloudFunctionVpcEgress: {
           enum: ['ALL', 'ALL_TRAFFIC', 'PRIVATE', 'PRIVATE_RANGES_ONLY'],
         },
+        cloudFunctionBuildEnvironmentVariables: {
+          type: 'object',
+          patternProperties: {
+            '^.*$': { type: 'string' },
```

Comment:
Should that regex really be this permissive? It accepts white spaces, etc, which I'm guessing might be not valid properties here.

Reply:
I haven't got permission to push to @shekhirin 's fork, but I believe a sane regex would be `[a-zA-Z_][a-zA-Z0-9_]*` based on this answer and the first comment on it https://stackoverflow.com/a/2821183/9997939